packages/govern-subgraph/schema.graphql

```diff
@@ -16,28 +16,18 @@ type Govern @entity {
   id: ID!
   address: Bytes!
   metadata: Bytes
-  games: [OptimisticGame!] @derivedFrom(field: "executor")
-  executions: [Execution!]
+  correspondingEntries: [RegistryEntry!] @derivedFrom(field: "executor")
+  containers: [Container]!
   roles: [Role!]
 }
 
-type Execution @entity {
-  id: ID!
-  queue: GovernQueue!
-  packet: QueuePacket!
-  results: [Bytes!]
-  createdAt: BigInt!
-}
-
 type GovernQueue @entity {
   id: ID!
   address: Bytes!
-  config: Config
-  games: [OptimisticGame!] @derivedFrom(field: "queue")
-  scheduled: [Schedule!]
-  challenged: [Challenge!]
-  vetoed: [Veto!]
-  executed: [Execution!] @derivedFrom(field: "queue")
+  config: Config!
+  correspondingEntries: [RegistryEntry!] @derivedFrom(field: "queue")
+  queued: [Container!]
+  challenged: [Challenge]!
```

Comment:
I might be wrong, but I think we can use `[E!]!` most of the time (non-null arrays of non-null entries)?

```suggestion
  correspondingEntries: [RegistryEntry!]! @derivedFrom(field: "queue")
  queued: [Container!]!
  challenged: [Challenge!]!
```